circuit/src/rns/secp256k1.rs

```diff
@@ -85,36 +85,56 @@ impl RnsParams<Fp, Fr, 4, 68> for Secp256k1_4_68 {
 	}
 
 	fn to_add_x() -> [Fr; 4] {
-		let limb0 = Fr::from_u128(252992975607365614304);
-		let limb1 = Fr::from_u128(246373929609370463456);
-		let limb2 = Fr::from_u128(147045207087024714823);
-		let limb3 = Fr::from_u128(661255989794087);
+		let limb0 =
+			Fr::from_u128(0x00000000000000000000000000000000000000000000000a30efcc2747d66b3d);
+		let limb1 =
+			Fr::from_u128(0x00000000000000000000000000000000000000000000000871230a93e8f70fec);
+		let limb2 =
+			Fr::from_u128(0x0000000000000000000000000000000000000000000000032aa8e8ae3d48867b);
+		let limb3 =
+			Fr::from_u128(0x000000000000000000000000000000000000000000000000000651634b333363);
```

Comment:
Why were these updated?

Reply:
I had a bug in `make_mul_aux` function. I edited it so the values changed.